ai.py

```diff
@@ -0,0 +1,9 @@
+from board import BoardState, Board
+
+class Ai:
+    def make_move(current_board_state):
+        new_state = current_board_state
```

Comment:
I don't understand this variable assignment. why not just use `current_board_state`